docs/how-to-guides/sync-dicom-metadata-to-fhir.md

```diff
@@ -1,12 +1,90 @@
 # Sync Medical Imaging Server for DICOM metadata into Azure API for FHIR Resources
 
+In this How-to Guide, you will learn how to sync Medical Imaging Server for DICOM metadata with FHIR. To do this, you will learn how to enable DICOM Cast by authentication with Managed Identity.
+
 For healthcare organizations seeking to integrate clinical and imaging data through the FHIR® standard, DICOM Cast enables synchronizing of DICOM image changes to FHIR&trade; ImagingStudy resource. This allows you to sync DICOM studies, series and instances into the FHIR&trade; ImagingStudy resource.
 
+Once you have competed the [prerequisites](##Prerequisites) and [enabled authentication](##Configure-Authentication-using-Managed-Identity) between your Medical Imaging Server for DICOM, your FHIR Server and your DICOM Cast deployment, you have enabled DICOM Cast. When you upload files to your Medical Imaging Server for DICOM, the corresponding FHIR resources will be persisted in your FHIR server.
+
 To learn more about DICOM Cast, see the [DICOM Cast Concept](../docs/Concepts/dicom-cast.md).
 
-## Deploy DICOM Cast
+## Prerequisites
+
+To enable DICOM Cast, you need to complete the following steps:
+
+1. [Deploy a Medical Imaging Server for DICOM](../quickstarts/deploy-via-azure.md)
+1. [Deploy a FHIR Server](https://github.com/microsoft/fhir-server)
+1. [Deploy DICOM Cast](../quickstarts/deploy-dicom-cast.md)
+
+## Configure Authentication using Managed Identity
+
+Currently there are three types of authentication supported for both Azure API for FHIR and Medical Imaging Server for DICOM: Managed Identity, OAuth2 Client Credential and OAuth2 User Password. The authentication can be configured via the application settings by the appropriate values in the `Authentication` property of the given server. For details on the three types, see [DICOM Cast authentication](/converter/dicom-cast/docs/authentication.md).
+
+This section will provide an end to end guide for configuring authentication with Managed Identity.

```

Comment:
Maybe I am not understanding this. I am assuming this documentation is to support MI for dicom-cast. The ACI already has the system assigned MI. That should be given access to Fhir and Dicom to use MI, correct?

Reply:
Thanks @smithago I didn't realize ACI already has MI assigned, I thought this was required to enable MI. @jackliums could you please take a look at this? Thanks!

Reply:
So the general flow would be 

1. Deploy the DICOM Server and FHIR Server
1. Deploy DICOM Cast (the template defaults to having managed identity enabled on the ACI)
1. Update your auth in the DICOM Server or FHIR server to allow access to the MI
1. Update your DICOM Cast configuration to use the managed identity